Index: Program.cs
===================================================================
--- Program.cs	(revision 77364)
+++ Program.cs	(working copy)
@@ -1,8 +1,6 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Windows.Forms;
-using Ninject;
+using OpenUO.Core.Patterns;
 
 namespace Ultima.Winforms.Sample
 {
@@ -14,14 +12,16 @@
         [STAThread]
         static void Main()
         {
-            IKernel kernel = new StandardKernel();
+            IoCContainer container = new IoCContainer();
 
-            kernel.Load<OpenUO.Ultima.UltimaSDKCoreModule>();
-            kernel.Load<OpenUO.Ultima.Windows.Forms.UltimaSDKBitmapModule>();
+            container.RegisterModule<OpenUO.Ultima.UltimaSDKCoreModule>();
+            container.RegisterModule<OpenUO.Ultima.Windows.Forms.UltimaSDKBitmapModule>();
 
+            container.Register<SampleForm>();
+
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(kernel.Get<SampleForm>());
+            Application.Run(container.Resolve<SampleForm>());
         }
     }
 }
